rest/handler.go

```diff
@@ -1109,10 +1109,13 @@ func (h *handler) readSanitizeJSON(val interface{}) error {
 	}
 
 	// Expand environment variables.
-	content, err = expandEnv(h.ctx(), content)
-	if err != nil {
-		return err
+	if !base.BoolDefault(h.server.Config.Unsupported.DisallowDbConfigEnvVars, false) {
```

Comment:
This usage makes me feel like naming this as AllowDbConfigEnvVars, and setting the default to true, is easier to parse:
```
if base.BoolDefault(h.server.Config.Unsupported.AllowDbConfigEnvVars, true) {
```
but I can live with it if you don't agree.